compiler/shared_ast/optimizations.ml

```diff
@@ -184,7 +184,7 @@ let rec optimize_expr :
       when name = name1 ->
       Mark.remove (StructField.Map.find field fields)
     | EDefault { excepts; just; cons } -> (
-      (* TODO: mechanically prove each of these optimizations correct :) *)
+      (* TODO: mechanically prove each of these optimizations correct *)
```

Comment:
:(

Reply:
(my editor was'nt clever enough to pair the parens of the comment regardless of the emoji… now I feel bad for killing an innocent smile :cry: )